### PR TITLE
[Don't merge]  Trigger builds for MAGMA with cuda11.3 on Windows

### PR DESCRIPTION
**Pull request by mszhanyi**
Fixes #{issue number}

### CODE REVIEW

File: .circleci/scripts/magma_build.bat [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+:: Follow the installer steps and install MKL to default path
+
+pushd %CD%
+call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+REM call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+::call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\x86_amd64\vcvarsx86_amd64.bat"
+popd
+
+if not exist magma (
+  git clone https://github.com/peterjc123/magma.git magma
```
Comment (janeyx99) [inline .circleci/scripts/magma_build.bat:38]: Should we use this magma vs the one in http://icl.utk.edu/projectsfiles/magma/downloads/?

File: .circleci/scripts/magma_build.bat [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+cmake --build . --target install --config %CONFIG% -- -j%NUMBER_OF_PROCESSORS%
+if errorlevel 1 exit /b 1
+
+cd ..\..\..
+
+:: Create 
+7z a magma_%MAGMA_VERSION%_cuda%CUVER_NODOT%_%CONFIG_LOWERCASE%.7z %cd%\magma_cuda%CUVER_NODOT%\magma\install\*
+
+rmdir /S /Q magma_cuda%CUVER_NODOT%\
+@endlocal
```
Comment (janeyx99) [inline .circleci/scripts/magma_build.bat:101]: Please add the missing new line 😛

File: build.bat [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@echo on
+
+call .circleci\scripts\cuda_install.bat
+if errorlevel 1 exit /b 1
+
+mkdir magma_build
+cd magma_build
+call ..\.circleci\scripts\magma_build.bat
+if errorlevel 1 exit /b 1
```
Comment (janeyx99) [inline build.bat:9]: Please add the missing new line 😛

**Review by janeyx99 — COMMENTED** [review]
I agree with @peterjc123 this should be in the builder repo (I'm planning on writing a GHA that uses these scripts for it today!). Thanks for adding the code for 11.3 :D

File: .circleci/scripts/magma_build.bat (hunk 1 above)
Comment (peterjc123) [reply .circleci/scripts/magma_build.bat:38]: It is just the package you mentioned with the patches in pytorch/builder.

File: .circleci/scripts/magma_build.bat (hunk 2 above)
Comment (peterjc123) [reply .circleci/scripts/magma_build.bat:101]: I suppose this is not the PR to be merged to master.

Comment (janeyx99) [reply .circleci/scripts/magma_build.bat:101]: Yea I'm moving these files to builder actually

**Pull request closed without merging**